integration-tests/observability-lgtm/src/test/java/io/quarkus/observability/test/LgtmTestBase.java

```diff
@@ -13,15 +13,15 @@
 public abstract class LgtmTestBase {
     private final Logger log = Logger.getLogger(getClass());
 
-    @ConfigProperty(name = "quarkus.grafana.url")
-    String url;
+    @ConfigProperty(name = "grafana.endpoint")
+    String endpoint;
 
     @Test
     public void testTracing() {
         log.info("Testing Grafana ...");
         String response = RestAssured.get("/api/poke?f=100").body().asString();
         log.info("Response: " + response);
-        GrafanaClient client = new GrafanaClient("http://" + url, "admin", "admin");
+        GrafanaClient client = new GrafanaClient(endpoint, "admin", "admin");
```

Comment:
Does this work w/o `http://` ?

Reply:
Not sure, I'm just trying to make things compatible with the previous version.

Reply:
Did you check / fix this?

Reply:
the endpoint already includes the `http://` part